### PR TITLE
Multi-cluster support: Added skeleton code for computes endpoints

### PR DESCRIPTION
**Pull request by dhruvsgarg**
Any cluster running flame is referred to as `compute` (consisting of one or several physical nodes). These different `computes` can be identified by their deployer node's `deployerId`.

This PR adds the required `yml` and openapi compiled files (with skeleton code) for creating REST API endpoints for computes to be registered, updated, deleted, or for listing their current status.

### CODE REVIEW

**Review by myungjin — COMMENTED** [review]
two comments were left. Please consider to address them.

File: api/computes_components.partials.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  type: object
+  properties:
+    deployerId:
+      type: string
+    state:
+      $ref: '#/components/schemas/ComputeState'
+    registeredAt:
+      type: string
+      format: date-time
+    startedAt:
```
[the remark this answers is not in the document]
Comment (dhruvsgarg) [reply api/computes_components.partials.yml]: Resolved

File: api/computes_components.partials.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+ComputeState:
+  enum:
+    - ready_compute
+    - starting_compute
+    - deploying_compute
+    - running_compute
+    - completed_compute
+    - terminating_compute
+    - terminated_compute
+    - failed_compute
```
[the remark this answers is not in the document]
Comment (dhruvsgarg) [reply api/computes_components.partials.yml]: Resolved

**Review by myungjin — APPROVED** [review]
lgtm

**Pull request merged**